paperqa/llms.py

```diff
@@ -283,6 +285,24 @@ class QdrantVectorStore(VectorStore):
     vector_name: str | None = Field(default=None)
     _point_ids: set[str] | None = None
 
+    def __init__(self, **data):
+        super().__init__(**data)
+
+    def __del__(self):
+        """Cleanup async client connection."""
+        try:
+            loop = asyncio.get_event_loop()
+            if loop.is_running():
+                loop.create_task(self.client.close())
+            else:
+                loop.run_until_complete(self.client.close())
+        except Exception:
+            pass
+
+    async def aclose(self):
+        """Explicitly close async client."""
+        await self.client.close()
```

Comment:
Can you update `client`'s `description` (about 22 lines up) to say `AsyncQdrantClient`